### PR TITLE
Fix static analyzer warnings

### PR DESCRIPTION
**Pull request by jack-stripe**
r? @bdorfman-stripe

### CODE REVIEW

File: Stripe/STPAddCardViewController.m [hunk 1]
```diff
@@ -71,10 +71,10 @@ @interface STPAddCardViewController ()<STPPaymentCardTextFieldDelegate, STPAddre
 @end
 
 static NSString *const STPPaymentCardCellReuseIdentifier = @"STPPaymentCardCellReuseIdentifier";
-static NSInteger STPPaymentCardEmailSection = 0;
-static NSInteger STPPaymentCardNumberSection = 1;
-static NSInteger STPPaymentCardBillingAddressSection = 2;
-static NSInteger STPPaymentCardRememberMeSection = 3;
+static const NSInteger STPPaymentCardEmailSection = 0;
```
Comment (bdorfman-stripe) [inline Stripe/STPAddCardViewController.m]: Is there a reason these are not just an enum and instead are separate static consts?

**Comment by jack-stripe** [comment]
re-r? @bdorfman-stripe

**Comment by jack-stripe** [comment]
Note that this also updates Travis' Xcode version to 8.1 (let me know if you can think of anything I missed that needs updating alongside that...)

**Comment by bdorfman-stripe** [comment]
Other than my one question, looks good, so feel free to merge if that change was intentional.

**Pull request merged**